README.md

```diff
@@ -15,7 +15,7 @@ Features
 
 phpDocumentor supports the following:
 
-* *PHP 5.3 compatible*, full support for Namespaces, Closures and more is provided.
+* *PHP 5.5 compatible*, full support for Namespaces, Closures and more is provided.
```

Comment:
We are still compatible with php 5.3 and higher code; you only need php 5.5 to run it